src/DIRAC/WorkloadManagementSystem/Agent/SiteDirector.py

```diff
@@ -55,10 +56,24 @@
 # dirac install file
 DIRAC_INSTALL = os.path.join(DIRAC.rootPath, 'DIRAC', 'Core', 'scripts', 'dirac-install.py')
 
-# status
-TRANSIENT_PILOT_STATUS = ['Submitted', 'Waiting', 'Running', 'Scheduled', 'Ready', 'Unknown']
-WAITING_PILOT_STATUS = ['Submitted', 'Waiting', 'Scheduled', 'Ready']
-FINAL_PILOT_STATUS = ['Aborted', 'Failed', 'Done']
+# pilot2 python files (NOT needed for pilot 3)
+DIRAC_PILOT = os.path.join(DIRAC.rootPath, 'DIRAC', 'WorkloadManagementSystem', 'PilotAgent', 'dirac-pilot.py')
+DIRAC_MODULES = [os.path.join(DIRAC.rootPath, 'DIRAC', 'WorkloadManagementSystem', 'PilotAgent', 'pilotCommands.py'),
+                 os.path.join(DIRAC.rootPath, 'DIRAC', 'WorkloadManagementSystem', 'PilotAgent', 'pilotTools.py')]
+
+PILOT_TRANSIENT_STATES = [PilotStatus.SUBMITTED,
+                          PilotStatus.WAITING,
+                          PilotStatus.RUNNING,
+                          PilotStatus.UNKNOWN,
+                          'Scheduled']
+
+PILOT_WAITING_STATES = [PilotStatus.SUBMITTED,
+                        PilotStatus.WAITING,
+                        'Scheduled']
+
+PILOT_FINAL_STATES = [PilotStatus.DONE,
+                      PilotStatus.ABORTED,
+                      PilotStatus.FAILED]
```

Comment:
Don't you want to move these to PilotStatus.py?

Reply:
I chose not to move these lists in `PilotStatus.py` because `PilotStatusAgent` and  `PilotManagerHandler` also contain such lists that are almost similar.
Do you think I should at least move the lists defined in `SiteDirector` to `PilotStatus`?

Reply:
I think we should have only one list, and `PilotStatus` looks like the best place for defining them.

Reply:
I factorized the different lists into 3 lists and moved them in `PilotStatus`